resources/views/welcome.blade.php

```diff
@@ -152,6 +152,21 @@
                 </div>
             </div>
         @endif
+        @if($clickupInvitePending)
+        <div class="col-sm-6 com-md-3 col-lg-4">
+                <div class="card">
+                    <div class="card-body">
+                        <h4 class="card-title">
+                            Clickup Invitation Pending
+                        </h4>
+                        <p class="card-text">
+                            You have been invited to the RoboJackets Clickup page. Please go to Clickup using the link below and accept your invitation.
```

Comment:
```suggestion
                            You have been invited to the RoboJackets ClickUp workspace. Please check your email to accept your invitation.
```

and include their ClickUp email address here so they know which email to check